nuxeo-core/nuxeo-core-bulk/src/main/java/org/nuxeo/ecm/core/bulk/BulkService.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * (C) Copyright 2018 Nuxeo (http://nuxeo.com/) and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Contributors:
+ *     Funsho David
+ */
+package org.nuxeo.ecm.core.bulk;
+
+import java.util.UUID;
+
+/**
+ * API to manage Bulk Operation Framework.
+ *
+ * @since 10.2
+ */
+public interface BulkService {
+
+    BulkStatus runOperation(BulkCommand command);
```

Comment:
Please add Javadoc to all methods.

Reply:
I'm not fond of the term "operation" you use here and elsewhere, it's confusing with Automation operations. Maybe "command" is enough?